src/screens/NoticeboardIndexScreen.tsx

```diff
@@ -0,0 +1,99 @@
+import { colors, texts } from '../config';
+import { graphqlFetchPolicy, parseListItemsFromQuery } from '../helpers';
+import { useStaticContent } from '../hooks';
+import { useRenderItem } from '../hooks/listHooks';
+import { NetworkContext } from '../NetworkProvider';
+import { getQuery, QUERY_TYPES } from '../queries';
+
+import { ListHeaderComponent } from './NestedInfoScreen';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+/* eslint-disable complexity */
+export const NoticeboardIndexScreen = ({ navigation, route }: StackScreenProps<any>) => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+  const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp });
+  const [refreshing, setRefreshing] = useState(false);
+
+  const consentForDataProcessingText = route.params?.consentForDataProcessingText ?? '';
+  const content = route.params?.content ?? '';
+  const query = route.params?.query ?? '';
+  const queryVariables = route.params?.queryVariables ?? {};
+  const subQuery = route.params?.subQuery ?? '';
+  const title = route.params?.title ?? '';
+
+  const { data, loading, refetch } = useQuery(getQuery(query), {
+    fetchPolicy,
+    variables: queryVariables
+  });
+  const listItems = parseListItemsFromQuery(query, data, consentForDataProcessingText, {
+    queryVariables
+  });
+
+  const {
+    data: dataHtml,
+    loading: loadingHtml,
+    refetch: refetchHtml
+  } = useStaticContent<string>({
+    name: content,
+    type: 'html',
+    skip: !content
+  });
+
+  const onRefresh = useCallback(async () => {
+    setRefreshing(true);
+    if (isConnected) {
+      await refetch?.();
+      await refetchHtml?.();
+    }
+    setRefreshing(false);
+  }, [isConnected, refetch]);
+
+  const renderItem = useRenderItem(QUERY_TYPES.PUBLIC_JSON_FILE, navigation);
+
+  if (loading && !data)
+    return (
+      <LoadingContainer>
+        <ActivityIndicator color={colors.accent} />
+      </LoadingContainer>
+    );
+
+  if (!data) {
+    return <EmptyMessage title={texts.noticeboard.emptyTitle} />;
+  }
+
+  const sectionData = [{ title: title, data: listItems }];
+
+  return (
+    <SafeAreaViewFlex>
+      <SectionList
+        refreshControl={
+          <RefreshControl
+            refreshing={refreshing}
+            onRefresh={onRefresh}
+            colors={[colors.accent]}
+            tintColor={colors.accent}
+          />
+        }
+        ListHeaderComponent={
+          <ListHeaderComponent
+            loading={loadingHtml}
+            html={dataHtml}
+            subQuery={subQuery}
+            navigation={navigation}
+          />
+        }
+        sections={sectionData}
+        renderSectionHeader={({ section: { title } }) => <SectionHeader title={title} />}
+        renderItem={renderItem}
+        keyExtractor={(item) => item.title + item.id}
+      />
+    </SafeAreaViewFlex>
+  );
+};
+/* eslint-enable complexity */
```

Comment:
![image](https://user-images.githubusercontent.com/1942953/202757901-1abc0c30-3d39-42dc-a078-4d0d2d646e2e.png)

the three headlines should not contain "anzeigen" as this makes user think to be pressable like a button.
i think we can entirely get rid of the headline for the list. the main headline is already present on the screens below the image, like on the initial sketches, right?!

![image](https://user-images.githubusercontent.com/1942953/202758248-76c6abbc-f016-4662-8681-27747ab82b4e.png)

Reply:
Yes, it looks like this in the drafts, but I used section list because I didn't like the look after I removed `SectionList`. I'll change it.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/698/commits/00a19f8f9ba995dcfbf34843f7404fcef1a7816f

![Simulator Screen Shot - iPhone 8 Plus - 2022-11-21 at 12 44 13](https://user-images.githubusercontent.com/11755668/203043333-11ab45fb-850c-415e-9ede-08bd34d70393.png)